### ISSUE TITLE
Disappearing logs

### ISSUE THREAD

**Issue by DamnDaniel72** [StrangeLoopGames/EcoIssues#15557]
When a log lies down to the ground after chopping and you exit the game, after restarting the log just is not there anymore.

Game Version: 0.8.3.3 beta

**Comment by SlayksWood** [comment]
1. Do you always have this issue?
2. Do you play in single player?
3. If you'll have this issue another time please provide log files.
Instruction is here https://github.com/StrangeLoopGames/EcoIssues/wiki/Getting-the-Eco-game-log-file

**Comment by DamnDaniel72** [comment]
Hello,
I'm sorry that I provided you with such marginal information.

1. Yesterday (05.04.2020) I first had this issue after running the game for the first time since February. I have been able to reproduce this issue every time. Today I tried it again (with video), but this time it seems to work just fine every time i tried. 

2. Yes, I am playing in single player

3. Here are the log files:
[Player.log](https://github.com/StrangeLoopGames/EcoIssues/files/4438188/Player.log)
[Player-prev.log](https://github.com/StrangeLoopGames/EcoIssues/files/4438189/Player-prev.log)
